app/searches/lookup_drugs.rb

```diff
@@ -29,8 +30,9 @@ def self.match_search_terms_to_objects(search_terms, scope)
     search_terms = search_terms.dup
     drug_results = DataModel::Drug.send(scope).where(name: search_terms)
     search_terms = search_terms - drug_results.map(&:name)
-    drug_alias_results = DataModel::DrugClaimAlias.send(scope).where(alias: search_terms)
+    drug_alias_results = DataModel::DrugAlias.send(scope).where(alias: search_terms)
     search_terms = search_terms - drug_alias_results.map(&:alias)
+    #Can we get rid of this case
```

Comment:
I'm confused. You say "I don't think so?" but the rest of your comment sounds like we can indeed get rid of this case to search against the claim.